Python/ceval.c

```diff
@@ -747,24 +747,19 @@ _PyEval_InitRuntimeState(struct _ceval_runtime_state *ceval)
 #endif
 }
 
-int
-_PyEval_InitState(struct _ceval_state *ceval)
+void
+_PyEval_InitState(struct _ceval_state *ceval, PyThread_type_lock pending_lock)
```

Comment:
Would this be a good opportunity to remove this function?
The `_pending_calls` struct belongs on the interpreter state, not on the `ceval` as only the main interpreter can handle them.

Reply:
There's enough going on in this PR that I'd rather not.